src/hotkeys.cpp

```diff
@@ -24,7 +24,7 @@ char *_hotkeys_file;
  * List of all HotkeyLists.
  * This is a pointer to ensure initialisation order with the various static HotkeyList instances.
  */
-static SmallVector<HotkeyList*, 16> *_hotkey_lists = NULL;
+static std::vector<HotkeyList*> *_hotkey_lists = NULL;
```

Comment:
pointer to a vector of pointers... lovely. I see from the comment that it was for initialisation order reasons, but is that still necessary?